widgetastic_manageiq/__init__.py

```diff
@@ -5374,5 +5374,61 @@ def select(self, name):
         return False
 
 
+class RolesSelector(Widget):
+    """This widget for custom button role selection"""
+
+    ROOT = "//div[@class='col-md-10']//table"
+    CELLS = "./tbody/tr/td"
+    INPUTS = "./tbody/tr/td/input"
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent=parent, logger=logger)
+
+    @property
+    def _elements(self):
```

Comment:
I'd like to give this property a more descriptive name, like `_cell_input_map`, since this isn't simply element objects.

Reply:
thanks, it will give a meaningful name to non-user property